gf-webapp/src/logic/aboutUsPage/serviceBlock/ServiceBlock.module.scss

```diff
@@ -26,6 +26,15 @@
   overflow-y: scroll;
 }
 
+.textLeft {
+  color: $color-text;
+  font-size: 18px;
+  font-family: Montserrat Regular;
+  margin-bottom: 30px;
+  margin-left: 0;
+  overflow-y: scroll;
+}
```

Comment:
All paddings and margins and other propertis will be moved out in conts in task #75